tests/model_serving/model_server/kserve/negative/test_rapid_isvc_create_delete.py

```
@@ -0,0 +1,258 @@
+
+import pytest
+import structlog
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.namespace import Namespace
+from ocp_resources.secret import Secret
+from ocp_resources.serving_runtime import ServingRuntime
+from pytest_testconfig import config as py_config
+from timeout_sampler import TimeoutSampler
+
+from tests.model_serving.model_server.kserve.negative.constants import INVALID_S3_ACCESS_KEY, INVALID_S3_SIGNING_KEY
```

Flake8 (7.3.0)

[error] 'tests.model_serving.model_server.kserve.negative.constants.INVALID_S3_ACCESS_KEY' imported but unused

(F401)